modules/claims_api/spec/lib/claims_api/v2/disability_compensation_pdf_mapper_spec.rb

```diff
@@ -309,6 +309,88 @@
         expect(multi_exp_location).to eq('Guam')
         expect(multi_exp_hazard).to eq('RADIATION')
       end
+
+      it 'maps herbicide correctly when dates are not included' do
+        form_attributes['toxicExposure']['herbicideHazardService']['serviceDates'] = nil
+
+        mapper.map_claim
+
+        toxic_exp_data = pdf_data[:data][:attributes][:exposureInformation][:toxicExposure]
+        herb_service_dates = toxic_exp_data[:herbicideHazardService][:serviceDates]
+
+        expect(herb_service_dates).to eq(nil)
+      end
+
+      it 'maps additional exposures correctly when dates are not included' do
+        form_attributes['toxicExposure']['additionalHazardExposures']['exposureDates'] = nil
+
+        mapper.map_claim
+
+        toxic_exp_data = pdf_data[:data][:attributes][:exposureInformation][:toxicExposure]
+        additional_exposure_dates = toxic_exp_data[:additionalHazardExposures][:exposureDates]
+
+        expect(additional_exposure_dates).to eq(nil)
+      end
+
+      context '526 section 4, herbicideHazardService exposures null data' do
```

Comment:
From here down to line 393 is jsut a refactor.  These tests were labelled as being Section 5 and were placed in section 5 but are for section 4.  The tests themselves were not changed at all, just moved into the proper 'area'